visualization/app/codeCharta/util/layoutHelper.ts

```diff
@@ -160,4 +160,36 @@ export class TreeMapHelper {
 			}
 		}
 	}
+
+	public static calculateSize(node: CodeMapNode, metricName: string) {
+		let totalSize = node.attributes[metricName] || 0
+
+		if (totalSize === 0 && node.children && node.children.length > 0) {
+			for (const child of node.children) {
+				totalSize += LayoutHelper.calculateSize(child, metricName)
+			}
+		}
+		return totalSize
+	}
```

Comment:
Are you aware of that for folder nodes, the sum of its children is set inside the `NodeDecorator.decorateParentNodesWithAggregatedAttributes()`)? The nodes contain that sum/median depending on the attributeType.